CHANGELOG.md

```diff
@@ -1,3 +1,59 @@
+## Fleet 4.0.0 RC2 (Jun 18, 2021)
+
+The primary additions in Fleet 4.0.0 are the new Role-based access control (RBAC) and Teams features. 
+
+RBAC adds the ability to define a user's access to features in Fleet. This way, more individuals in an organization can utilize Fleet with appropriate levels of access.
+
+* Check out the [permissions documentation](https://github.com/fleetdm/fleet/blob/5e40afa8ba28fc5cdee813dfca53b84ee0ee65cd/docs/1-Using-Fleet/8-Permissions.md) for a breakdown of the new user roles.
+
+Teams adds the ability to separate hosts into exclusive groups. This way, users can easily act on consistent groups of hosts. 
+
+* Read more about the Teams feature in [the documentation here](https://github.com/fleetdm/fleet/blob/5e40afa8ba28fc5cdee813dfca53b84ee0ee65cd/docs/1-Using-Fleet/9-Teams.md).
+
+### New features breakdown
+
+* Add ability to define a user's access to features in Fleet by introducing the Admin, Maintainer, and Observer roles. Available in Fleet Core.
+
+* Add ability to separate hosts into exclusive groups with the Teams feature. The Teams feature is available for Fleet Basic customers. Check out the list below for the new functionality included with Teams:
+
+* Teams: Add ability to enroll hosts to one team using team specific enroll secrets.
+
+* Teams: Add ability to manually transfer hosts to a different team in the Fleet UI.
+
+* Teams: Add ability to apply unique agent options to each team. Note that "osquery options" have been renamed to "agent options."
+
+* Teams: Add ability to grant users access to one or more teams. This allows you to define a user's access to specific groups of hosts in Fleet.
+
+* Add ability to create an API-only user. API-only users cannot access the Fleet UI. These users can access all Fleet API endpoints and `fleetctl` features. Available in Fleet Core.
+
+* Add Redis cluster support. Available in Fleet Core.
+
+* Fix a bug that prevented the columns chosen for the "Hosts" table from persisting after logging out of Fleet.
+
+### Upgrade plan
+
+Fleet 4.0.0 is a major release and introduces several breaking changes and database migrations. 
+
+* Use strictly `fleet` in Fleet's configuration, API routes, and environment variables. This means that you must update all usage of `kolide` in these items. The backwards compatibility introduced in Fleet 3.8.0 is no longer valid in Fleet 4.0.0.
```

Comment:
```suggestion
* Use strictly `fleet` in Fleet's configuration, API routes, and environment variables. Users must update all usage of `kolide` in these items (deprecated since Fleet 3.8.0).
```